.github/workflows/live-vrf-tests.yml

```diff
@@ -0,0 +1,191 @@
+name: Live VRF Tests
+on:
+  workflow_dispatch:
+    inputs:
+      slack_user_id:
+        description: "The Slack member ID to notify"
+        required: true
+        type: string
+      networks:
+        description: "Remove any networks you don't want to run tests on"
+        required: true
+        default: "SEPOLIA,OPTIMISM_SEPOLIA,ARBITRUM_SEPOLIA"
+      test:
+        description: "Choose test you want to run"
+        required: true
+        type: choice
+        options:
+          - "TestVRFBasic"
+          - "TestVRFv2Basic"
+          - "TestVRFv2Plus"
+
+env:
+  CHAINLINK_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
+  INTERNAL_DOCKER_REPO: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com
+  MOD_CACHE_VERSION: 2
+  CHAINLINK_NODE_FUNDING: .5
+  PYROSCOPE_KEY: ${{ secrets.QA_PYROSCOPE_KEY }}
+  LOKI_TENANT_ID: ${{ vars.LOKI_TENANT_ID }}
+  LOKI_URL: ${{ secrets.LOKI_URL }}
+  LOKI_BASIC_AUTH: ${{ secrets.LOKI_BASIC_AUTH }}
+  LOGSTREAM_LOG_TARGETS: loki
+  GRAFANA_URL: ${{ vars.GRAFANA_URL }}
+  RUN_ID: ${{ github.run_id }}
+
+  CHAINLINK_COMMIT_SHA: ${{ github.sha }}
+  CHAINLINK_ENV_USER: ${{ github.actor }}
+  TEST_LOG_LEVEL: debug
+
+jobs:
+
+    # Build Test Dependencies
+
+  build-chainlink:
```

Comment:
I dont think i need this step, or maybe it could be optional - I would like to specify image in the toml config